client/components/site-title/index.jsx

```diff
@@ -21,13 +22,15 @@ class SiteTitleControl extends React.Component {
 		blogname: PropTypes.string,
 		blogdescription: PropTypes.string,
 		disabled: PropTypes.bool,
+		isTitleRequired: PropTypes.bool,
```

Comment:
This prop name is inconsistent with `autoFocusBlogname` (and arguably with `blogname`). I wouldn't say `blogname` is necessarily better, but props that refer to the same thing should use the same term IMO...